data/tribes/buildings/militarysites/amazons/treetop_sentry/init.lua

```diff
@@ -38,7 +38,7 @@ wl.Descriptions():new_militarysite_type {
 
    messages = {
       -- TRANSLATORS: Message sent by an Amazon military site
-      occupied = pgettext("amazons_building", "Your soldiers have occupied your treetop sentry."),
```

Comment:
while translating this I found that plural would have suited better here as well although only one of my soldiers occupies it. Reason is that that the plural is synonym for the whole army imho. Translating it to singular would mean I have only one soldier at all in this context. So to not change every string here I'd rather vote to turn this back into the plural state